.travis.yml

```diff
@@ -0,0 +1,55 @@
+language: php
+
+branches:
+  only:
+    - master
+    - feat/first
+
+matrix:
+  include:
+    - php: 5.3
```

Comment:
We are going to support PHP 5.3?

Reply:
For the moment we've not discuss about it, I decided to put it, but if we no longer support PHP 5.3 I'll remove it.

Reply:
I prefer we just start working assuming that we are going to use PHP 5.6. If you agree, of course. That is the most likely case IMO.

Reply:
Done